recipe/meta.yaml

```diff
@@ -61,10 +59,8 @@ about:
   summary: Ordination methods, diversity analysis and other functions for community and vegetation
     ecologists.
   license_family: GPL2
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
 
 extra:
   recipe-maintainers:
-    - daf
-    - ocefpaf
```

Comment:
It is OK to remove me, I have universal access as a core member anyway, but I don't believe everybody you removed are in the `conda-forge/r` team.

Reply:
I'm fine being removed as well but am happy to stay on if needed.  All I usually do is merge when CI looks happy.

Reply:
@ocefpaf I forgot to check that. In any case, it seems that only you and @daf are not part of `conda-forge/r`.